pkg/controller/scyllacluster/sync_certs.go

```diff
@@ -0,0 +1,306 @@
+package scyllacluster
+
+import (
+	"context"
+	"crypto/x509/pkix"
+	"fmt"
+	"net"
+	"sort"
+	"strings"
+	"time"
+
+	scyllav1 "github.com/scylladb/scylla-operator/pkg/api/scylla/v1"
+	"github.com/scylladb/scylla-operator/pkg/controllerhelpers"
+	ocrypto "github.com/scylladb/scylla-operator/pkg/crypto"
+	"github.com/scylladb/scylla-operator/pkg/helpers"
+	"github.com/scylladb/scylla-operator/pkg/internalapi"
+	okubecrypto "github.com/scylladb/scylla-operator/pkg/kubecrypto"
+	"github.com/scylladb/scylla-operator/pkg/naming"
+	"github.com/scylladb/scylla-operator/pkg/resourceapply"
+	"github.com/scylladb/scylla-operator/pkg/scheme"
+	cqlclientv1alpha1 "github.com/scylladb/scylla-operator/pkg/scylla/api/cqlclient/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	apierrors "k8s.io/apimachinery/pkg/util/errors"
+	"k8s.io/klog/v2"
+)
+
+func makeScyllaConnectionConfig(
+	sc *scyllav1.ScyllaCluster,
+	secrets map[string]*corev1.Secret,
+	configMaps map[string]*corev1.ConfigMap,
+) (*corev1.Secret, error) {
+	clientCertSecretName := naming.GetScyllaClusterLocalUserAdminCertName(sc.Name)
+	clientCertSecret, found := secrets[clientCertSecretName]
+	if !found {
+		return nil, fmt.Errorf("secret %q doesn't exist or is not own by this object", naming.ManualRef(sc.Namespace, clientCertSecretName))
+	}
+
+	clientCertsBytes, clientKeyBytes, err := okubecrypto.GetCertKeyDataFromSecret(clientCertSecret)
```

Comment:
it's the other way around, GetCertKey returns Key and Cert. Maybe name should be adjusted to the order of return values. 
Otherwise I tested the generated bundle and it works.

Reply:
thanks